spec/models/log_entry_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe LogEntry do
+  describe 'entry_time' do
+    it 'returns a string of created_at date and time' do
+      time = '10:30 am, Thursday, March 16, 2017'
+      Timecop.freeze(Time.local(2017, 0o3, 16, 10, 30, 0)) do 
```

Comment:
Yeah, I wish Rubocop would be more specific that you only want to put the `o` in there if you *really want* an octal number.  Hint: no one ever really wants an octal number.